tools/copy_engine_content.py

```diff
@@ -7,11 +7,10 @@
 import shutil
 import spear
 
-
 if __name__ == "__main__":
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--unreal_engine_dir", required=True)
+    parser.add_argument("--unreal_engine_dir", default=r"F:\Program Files\Epic Games\UE_5.5")
```

Comment:
This is not a meaningful default for anyone besides you. It is user-specific and Windows-specific. Please revert.